docs/running-on-mesos.md

```diff
@@ -387,6 +387,14 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.webui.url</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    Set the Spark Mesos framework's webui_url pointing to a web-based interface for interacting with the framework.
```

Comment:
s/pointing/to point

"to point a web-based interface for interacting with the framework" is a bit redundant.  we can probably remove it.

Not a blocker